recipe/migrations/cuda118.yaml

```diff
@@ -26,31 +26,39 @@ __migrator:
       - quay.io/condaforge/linux-anvil-ppc64le            # [os.environ.get("BUILD_PLATFORM") == "linux-ppc64le"]
       - quay.io/condaforge/linux-anvil-armv7l             # [os.environ.get("BUILD_PLATFORM") == "linux-armv7l"]
+
+      # Legacy CUDAs
       - quay.io/condaforge/linux-anvil-cuda:9.2           # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:10.0          # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:10.1          # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:10.2          # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:11.0          # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:11.1          # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+
+      # CUDA 11.2
       - quay.io/condaforge/linux-anvil-cuda:11.2          # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
-      # case: native compilation (build == target)
+      # CUDA 11.2 arch: native compilation (build == target)
       - quay.io/condaforge/linux-anvil-ppc64le-cuda:11.2  # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-ppc64le"]
       - quay.io/condaforge/linux-anvil-aarch64-cuda:11.2  # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-aarch64"]
-      # case: cross-compilation (build != target)
+      # CUDA 11.2 arch: cross-compilation (build != target)
       - quay.io/condaforge/linux-anvil-cuda:11.2          # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:11.2          # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+
+      # CUDA 11.8
       - quay.io/condaforge/linux-anvil-cuda:11.8          # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
-      # case: native compilation (build == target)
+      # CUDA 11.8 arch: native compilation (build == target)
       - quay.io/condaforge/linux-anvil-ppc64le-cuda:11.8  # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-ppc64le"]
       - quay.io/condaforge/linux-anvil-aarch64-cuda:11.8  # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-aarch64"]
-      # case: cross-compilation (build != target)
+      # CUDA 11.8 arch: cross-compilation (build != target)
       - quay.io/condaforge/linux-anvil-cuda:11.8          # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:11.8          # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
-      # case: non-CUDA builds
+
+      # non-CUDA arch: cross-compilation (build != target)
       - quay.io/condaforge/linux-anvil-cos7-x86_64        # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
```

Comment:
Comments look good in general, but this last one is definitely not cross-compilation. I also don't know why we even have non-CUDA at the end again, given that this should be covered by the lines at the beginning.

Reply:
It seems we're asking the same question (https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4834#discussion_r1312659535) 🙂

Reply:
Fixed the comment. Sorry about that

It appears to be last in the CUDA 12.0 migrator where this was taken from

https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/33db02d50ade2f110a9420d5600dc8d258576a58/recipe/migrations/cuda120.yaml#L46-L47

Reply:
I know that this is taken from the CUDA 12 one, but I didn't get why it was necessary. In any case, not a blocker.